Sources/Maintenance/Migration/v3_0/UserReactions.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+/**
+ * Simple Machines Forum (SMF)
+ *
+ * @package SMF
+ * @author Simple Machines https://www.simplemachines.org
+ * @copyright 2024 Simple Machines and individual contributors
+ * @license https://www.simplemachines.org/about/smf/license.php BSD
+ *
+ * @version 3.0 Alpha 3
+ */
+
+declare(strict_types=1);
+
+namespace SMF\Maintenance\Migration\v3_0;
+
+use SMF\Db\DatabaseApi as Db;
+use SMF\Maintenance\Migration\MigrationBase;
+use SMF\Db\Schema\v3_0;
+use SMF\Db\Schema\v3_0\UserReacts;
+
+class UserReactions extends MigrationBase
+{
+	/*******************
+	 * Public properties
+	 *******************/
+
+	/**
+	 *
+	 */
+	public string $name = 'Converting likes to reactions';
+
+	/****************
+	 * Public methods
+	 ****************/
+
+	/**
+	 *
+	 */
+	public function execute(): bool
+	{
+		// Make sure we haven't already done this...
+		$cols = Db::$db->list_columns('messages');
+		
+		// If the reactions column exists in the messages table, there's nothing to do
+		if(!in_array('reactions', $cols))
+		{
+			// Does the user_likes table exist?
+			$table_exists = Db::$db->list_tables(false, '%user_likes');
+			if (!empty($table_exists))
+			{
+				// Rename the table
+				Db::$db->query('', 'ALTER TABLE {db_prefix}user_likes RENAME TO {db_prefix}user_reacts');
+				
+				// Add the new column
+				Db::$db->add_column('{db_prefix}user_reacts', ['name' => 'id_reaction', 'type' => 'smallint', 'not_null' => true, 'default' => '0']);
```

Comment:
Ensure that you have a use statement to `UserReacts` and you could do:
```
$tbl = new UserReacts();

Db::$db->add_column('{db_prefix}user_reacts', $tbl->columns['id_reaction']);
```

Thus ensuring that you are asking it to do the changes as needed to match the schema.